runtime-modules/storage/Cargo.toml

```diff
@@ -73,15 +74,20 @@ default_features = false
 package = 'substrate-membership-module'
 path = '../membership'
 
+[dependencies.minting]
```

Comment:
when removing the add extra genesis block out of the `data_object_storage_registry` module we can remove some unnecessary dependencies from here? Or do we still need all the dependencies of `substrate-working-group-module` as well, ie. minting, stake etc..

Reply:
Moved mint module to the dev-dependencies. All of these modules mint, stake ... etc are required for mocking.

Reply:
oh right of course :+1: